ai-ml/trainium-inferentia/outputs.tf

```diff
@@ -1,4 +1,9 @@
-output "configure_kubectl" {
+/* output "configure_kubectl" {
   description = "Configure kubectl: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
   value       = "aws eks --region ${var.region} update-kubeconfig --name ${var.name}"
+} */
```

Comment:
what is the reason for this change? 
Could we remove the commented lines from the code?

Reply:
I was having issues using hardcoded eks cluster name. If I had to deploy another set of cluster I had issues because KMS would not allow creation of another AWS managed key with the same name. That is why I have used random string to the cluster name thats hardcoded.

Reply:
I will remove the commented lines